### DISCUSSION TITLE
Expanding a cluster with single-node Single Drive server pool with a Multi-Node Multi-Drive server pool.

### DISCUSSION THREAD

**Discussion by Jazz9** [minio/minio#19040] [Q&A] [answered]
Has anyone expanded a cluster of one server / one drive with a server pool multi-node multi drive?

It has an error saying you can't mix the types.

We are trying to add the new server pool and then decommission the old server pool. Upgrades 👯 

Jazz

**Comment by harshavardhana** [comment, marked as answer]
Not supported just `mc mirror` the content to the distributed setup.

**Reply by Jazz9** [reply]
Thankyou for the prompt response.
